CHANGELOG.md

```diff
@@ -4,6 +4,48 @@
 
 ### New features
 
+#### Check checkboxes by using the `checkedValues` option
+
+When using the Checkboxes nunjucks macro you can now use `checkedValues` option to indicate which checkboxes should be checked when the page loads.
+
+For example, this would check the checkboxes that have the values 'foo' and 'bar':
+
+```
+checkedValues: ['foo', 'bar']
+```
```

Comment:
Maybe this suggestion is overkill considering it would just be for one update, _but..._could we add a full example, plus the existing content, to a Google Doc, then ask a few frontend users for their thoughts?